R/utils.R

```diff
@@ -82,13 +81,13 @@
   invisible(NULL)
 }
 
-.checkStrata <- function(data, strata) {
+.checkStrata <- function(check_data, strata) {
```

Comment:
why renamed?

Reply:
I am avoiding any variables names that conflict with base R